packages/console/src/components/Table/index.tsx

```diff
@@ -88,9 +89,10 @@ const Table = <
                       onClickRow?.(row);
                     }}
                   >
-                    {columns.map(({ dataIndex, colSpan, className, render }) => (
-                      <td key={dataIndex} colSpan={colSpan} className={className}>
-                        {render(row[dataIndex], row)}
+                    {columns.map(({ colSpan, className, render }, index) => (
+                      // eslint-disable-next-line react/no-array-index-key
+                      <td key={`td-${index}`} colSpan={colSpan} className={className}>
```

Comment:
`td-${index}` can be conflict as each row will have a `td` that has the same index

Reply:
After offline discussion, we think the dataIndex is still needed, as it's the unique identifier of a column

Reply:
updated.